<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose a material name and a type of material and give its parameters. The mirror material is already added to the materials group.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditName">
     <property name="placeholderText">
      <string>material_name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabConstIndexGlass">
      <attribute name="title">
       <string>Constant Index Glass</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxIndex">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Constant Index</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabModelGlass">
      <attribute name="title">
       <string>Model Glass</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxN0">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.497496000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Conrady n0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_A">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.010099000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_B">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="singleStep">
          <double>0.000100000000000</double>
         </property>
         <property name="value">
          <double>0.000328000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Conrady A</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Conrady B</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGrinMedium">
      <attribute name="title">
       <string>Grin Medium</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>dn/dz(x, y, z)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_FOf_Nz"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_FOf_Nx"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>dn/dx(x, y, z)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="comboBox_FO_Nz"/>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="comboBox_FO_Ny"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_FOf_N"/>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_FOf_Ny"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBox_FO_N"/>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBox_FO_Nx"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>n(x, y, z)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>dn/dy(x, y, z)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Function Name in Function Object</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Function Object</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBoxCatalogue"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
